pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -97,6 +99,20 @@ func (r *Reconciler) ReconcileWorkerNodes(ctx context.Context, log logr.Logger,
 	return controller.Result{}, nil
 }
 
+// ValidateDatacenterConfig updates the cluster status if the TinkerbellDatacenter status indicates that the spec is invalid.
+func (r *Reconciler) ValidateDatacenterConfig(ctx context.Context, log logr.Logger, clusterSpec *c.Spec) (controller.Result, error) {
```

Comment:
Do we need any other validations on the ip itself?

Reply:
You mean this one? https://github.com/aws/eks-anywhere/blob/25e7072b79804fe1a057d268688a672a56bab815/pkg/providers/tinkerbell/reconciler/reconciler.go#L62

Reply:
This is tinkerbell ip right? Isn't it different?

Reply:
it is different yes